projects/swarmd/swarmd/Dockerfile

```diff
@@ -1,22 +1,45 @@
+FROM weaveworks/weave:e8fdf3e549348592f5802a730562e175e05bff8a63d99051440ebb933f6e7ef2 AS weave
+
+# Nothing to do in here, just for COPY --from=weave below
+
 FROM linuxkit/alpine:630ee558e4869672fae230c78364e367b8ea67a9 AS build
 
 RUN \
   apk update && apk upgrade && \
   apk add --no-cache \
+  bash \
   ca-certificates \
   gcc \
   git \
   go \
   libc-dev \
+  linux-headers \
   make \
   && true
 
 ENV GOPATH=/root/go
 
-# https://github.com/ijc25/swarmkit/tree/containerd-wip
+#####################################################################
+# Output filesystem skeleton
+
+RUN mkdir -p /out/etc/apk && cp -r /etc/apk/* /out/etc/apk/
+RUN apk add --no-cache --initdb -p /out \
+    alpine-baselayout \
+    busybox \
+    ca-certificates \
+    iptables \
+    musl \
+    util-linux
+
```

Comment:
you probably want to add a `RUN rm -rf /out/etc/apk /out/lib/apk /out/var/cache` here to remove the temporary `apk` files